### ISSUE TITLE
make_dataset.py - Wrong order of validation set and test set when splitting data

### ISSUE THREAD

**Issue by ralf-koenig** [jan-forest/autoencodix#8]
In `000_internal_config.yaml` a split of overall input data is defined such as:
`SPLIT: [0.7, 0.1, 0.2] # OPTIONAL Train, Validate, Test or "pre-computed`

Then first the 70% are split in `make_dataset.py` and assigned to training data, with 30% remaining records in `test_sample_set`.

Then in the second split of these 30% between validation and test data:

Wrong: 
```
                sample_list_test, sample_list_validate = train_test_split(
                    sample_list_test,
                    train_size=SPLIT[1] / (SPLIT[1] + SPLIT[2]),
                    random_state=random_state,
                )
```

This would assign 10% to test and 20% to validation.

Right:
```
                sample_list_validate, sample_list_test = train_test_split(
                    sample_list_test,
                    train_size=SPLIT[1] / (SPLIT[1] + SPLIT[2]),
                    random_state=random_state,
                )
```
                
This assigns 10% to validation and 20% to test.

In the code lines after this part, the general order `train,` `validate`, `test` should be kept to support better comprehension of the code.

**Comment by jan-forest** [comment]
Thanks for finding this mix up and code suggestion. We applied the fix. However, we keep the order in the code for now to avoid any additional errors by new changes.
